tests/scenarios/scenarios.go

```diff
@@ -238,8 +238,8 @@ func GetResponseStatusCode(resp []reflect.Value) (int, error) {
 }
 
 // SetCtx sets Go context in BDD context.
-func SetCtx(ctx gobdd.Context, value context.Context) {
-	ctx.Set(ctxKey{}, value)
+func SetCtx(ctx context.Context, bddCTX gobdd.Context) {
```

Comment:
I am not a big fan of this change as the rest of setters use BDD context as first argument.

Reply:
Yeah I just want to get rid of the lint noise.